src/ray/core_worker/transport/sequential_actor_submit_queue.cc

```diff
@@ -93,9 +93,7 @@ SequentialActorSubmitQueue::PopAllOutOfOrderCompletedTasks() {
 
 uint64_t SequentialActorSubmitQueue::GetSequenceNumber(
```

Comment:
maybe we can consider making `GetSequenceNumber` be a member function of task spec.